src/Backend/Modules/Pages/Pages.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+namespace Backend\Modules\Pages;
+
+use Symfony\Component\HttpKernel\Bundle\Bundle;
+
+class Pages extends Bundle
```

Comment:
Actually, this is required when using consoles, otherwise sometimes the module works and sometimes doesn't. It seems like the DependencyInjection doesn't get called otherwise.

Reply:
I added this to for "ContentBlock", "Location" and in the AppKernel.php. When removing that lines from AppKernel, the console command can't find those modules.

Reply:
Cause this is a no-go

Reply:
I have mentioned this a few times over the years that this is a problem.
Will see if I can't find a easy to replicate example.

Reply:
Well, it's very easy to reproduce.

Steps to reproduce:
1. Use any Fork CMS version.
1. Add `Backend\Modules\Location\DependencyInjection\LocationExtension`
```php
<?php

namespace Backend\Modules\Location\DependencyInjection;

use Symfony\Component\DependencyInjection\ContainerBuilder;
use Symfony\Component\Config\FileLocator;
use Symfony\Component\HttpKernel\DependencyInjection\Extension;
use Symfony\Component\DependencyInjection\Loader;

/**
 * This is the class that loads and manages your bundle configuration
 *
 * To learn more see {@link http://symfony.com/doc/current/cookbook/bundles/extension.html}
 */
class LocationExtension extends Extension
{
    public function load(array $configs, ContainerBuilder $container): void
    {
        echo "this is where I am";exit();
    }
}
```
1. Execute `bin/console cache:clear --no-warmup`
=> Nothing happens, that is not what we expect... It should output "this is where I am"

Reply:
there is an easy fix, look at how de make sure it is loaded on the site and make sure that also works in the console

Reply:
To let it work you need to add:
```php
<?php

namespace Backend\Modules\Location;

use Symfony\Component\HttpKernel\Bundle\Bundle;

class Location extends Bundle
{
}
```

and in AppKernel.php
`new \Backend\Modules\Location\Location(),`

Reply:
but that is not modular so not an acceptable solution :)

Reply:
Fixed